src/pyramid_debugtoolbar/panels/templates/session.dbtmako

```diff
@@ -0,0 +1,90 @@
+<h5>Session Configuration</h5>
+% if configuration:
+	<p>Using `ISessionFactory`: <code>${configuration}</code></p>
+% else:
+	<p>No `ISessionFactory` Configured</p>
+% endif
+
+
+## Panel Active
+% if is_active:
+	<p>This panel <span class="label label-success">was active</span> for this
+	request, and will always display the "Session" data -- even if `.session`
+	was not accessed during the request.
+	This behavior can be toggled with the Panel Settings.</p>
+% else:
+	<p>This panel <span class="label label-danger">was not active</span> for
+	this request, and will only display the "Session" data if `.session` was
+	accessed during the request.
+	This behavior can be toggled with the Panel Settings.</p>
+% endif
+
+
+<h5>Documentation Links</h5>
+<ul>
+	<li>API Docs:
+		<a href="http://docs.pylonsproject.org/projects/pyramid/en/latest/api/session.html">pyramid.session</a>
+	</li>
+	<li>Narrative Docs:
+		<a href="http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/sessions.html">Session</a>
+	</li>
+</ul>
+
+
+<h5>Session Accessed</h5>
+% if session_accessed["main"]:
+	<div class="alert alert-info">
+		`request.session` was accessed during the main `Request` handler.
+	</div>
+% else:
+	<div class="alert alert-warning">
+		`request.session` was not accessed during the main `Request` handler.
+	</div>
+% endif
+
+
+<h5>Session Data</h5>
+% if session_data["keys"]:
+	<table class="table table-striped table-condensed">
+		<thead>
+			<tr>
+				<th>key</th>
+				<th>ingress</th>
+				<th>egress</th>
+			</tr>
+		</thead>
+		<tbody>
+			<%
+				try:
+					keys = sorted(session_data["keys"])
+				except TypeError:
+					keys = sorted(session_data["keys"], key=lambda k: str(k))
+			%>
+			% for k in keys:
+				<tr>
+					<th>
+						% if k in session_data["changed"]:
+							<span class="label label-info">!</span>
```

Comment:
This icon alone does not tell the user what it means, and they would have to dig into this code to figure it out. Perhaps a tooltip or legend is appropriate here.

Reply:
tooltip!